### ISSUE TITLE
/GCCollectOnly captures CLR exception events

### ISSUE THREAD

**Issue by ryandle** [microsoft/perfview#1200]
I've been using /GCCollectOnly a lot recently and had issues with my buffer (2GB) filling up fairly quickly (a little over an hour.) I can of course make the buffer larger, but I was curious why it was happening because the GC events themselves are not very large.

It turns out, for the apps I'm profiling, these CLR exception events make up 92% of the total size:

> Microsoft-Windows-DotNETRuntime/Exception/Start
> Microsoft-Windows-DotNETRuntime/ExceptionCatch/Start
> Microsoft-Windows-DotNETRuntime/ExceptionFinally/Start
> Microsoft-Windows-DotNETRuntime/ExceptionFilter/Start

I believe this is because GCCollectOnly includes `ClrTraceEventParser.Keywords.Exception` [here](https://github.com/microsoft/perfview/blob/6e9043ace41ed0917cf566f387f4a7cce5e26a57/src/PerfView/CommandLineArgs.cs#L466):

```CS
ClrEvents = ClrTraceEventParser.Keywords.GC | ClrTraceEventParser.Keywords.Exception;
```

Is that intentional, or perhaps just a byproduct of copy/paste (as the comment above it suggests?)

Would removing the exception events from /GCCollectOnly be an acceptable change given how long its been this way? Or should I work around it by manually excluding these events in my own perfview commands?

(Happy to submit a PR if we think a change is the correct move.)

cc: @Maoni0

**Comment by Maoni0** [comment]
I'm totally ok with it.

**Comment by brianrob** [comment]
Fixed via #1201.
